### DISCUSSION TITLE
What exactly is a libyear?

### DISCUSSION THREAD

**Discussion by scottrassbach-utk** [ecoAPM/dotnet-libyear#167] [Q&A] [answered]
I ran libyear on a .net Framework 4.8.1 project, and it said it was 20.7 libyears behind.

What exactly does that quantify?

**Comment by SteveDesmond-ca** [comment, marked as answer]
1 libyear is the equivalent of a single dependency's installed version being released 1 year prior to the newest version, so your 20 libyear project could mean 5 dependencies that are 4 years behind each, or 10 that are 2 each, etc.

See [libyear.com](https://libyear.com) for more details!

**Reply by scottrassbach-utk** [reply]
Thanks!
